templates/netlify/remix.init/index.js

```diff
@@ -33,12 +34,32 @@ async function copyEdgeTemplateFiles(files, rootDirectory) {
   }
 }
 
+async function updatePackageJsonForEdge(filepath) {
+  let { dependencies, ...restOfPackageJson } = JSON.parse(
+    await fs.readFile(filepath, "utf-8")
+  );
+
+  delete dependencies["@remix-run/netlify"];
+  dependencies["@remix-run/netlify-edge"] = "*";
```

Comment:
At this point, `@remix-run/netlify` already has a specific version in `package.json`.

I would suggest to leave both `@remix-run/netlify` **and** `@remix-run/netlify-edge` in `package.json` & remove the one that's not needed here in `remix.init`, like I do in https://github.com/remix-run/indie-stack/pull/63, https://github.com/remix-run/blues-stack/pull/57 & https://github.com/remix-run/grunge-stack/pull/50

https://github.com/remix-run/indie-stack/blob/7c2506f60123e5242e6d1a47b9cdd7cfd638155d/remix.init/index.js#L140-L145